PROTOCOL.md

```diff
@@ -424,6 +429,64 @@ enough to get you started. You can watch the data sent and received from
 the server on a regular connection, or look at the client source code
 for a full list of message types.
 
+**Action requests**
+
+These are how the client sends the player's decisions to the server. All
+requests except `/undo` can be sent with `|RQID` at the end. `RQID` is
+`REQUEST.rqid` from `|request|`. Each `RQID` is a unique number used to
+identify which action the request was intended for and is used to protect
+against race conditions with the cancel button.
+
+If an invalid request is sent, the game will replace the missing or
+erroneous request with a valid choice, which is usually the first usable
+move. Similarly, if a switch was expected but a move was sent instead,
+the game will switch in a usable Pokemon.
+
+`|/team ORDER`
+
+> Chooses the team order. Numbers not listed are displaced to the back by swapping
+> them with the number that took their place. For example `/team 25` sets the team
+> order from default to 253416.
+
+`|/move NUMBER TARGET`
+
+> Uses your active Pokemon's `NUMBER`th move on `TARGET` Pokemon. `NUMBER` starts at
+> 1 and can be up to 24 (though usually 4).
```

Comment:
We might want to reverse the importance here.

> `NUMBER` is usually a number ranging from 1 to 4 (although it can range up to 24 in Custom Games where Pokémon have that many moves)